### ISSUE TITLE
Bug: Unknown provider: domeneshop

### ISSUE THREAD

**Issue by hestadgard** [qdm12/ddns-updater#884]
domeneshop is listed as a supported provider, but ddns-updater puts out error for domeneshop to be unknown

1. Is this urgent: Yes
2. DNS provider(s) you use: domeneshop
3. Program version:Running version 2.8.2 built on 2024-11-21T22:59:48Z (commit e95816ab468d89841a34f8f2cc6a44fe05cada3c)

4. What are you using to run the container: running binary on linux
5. Extra information (optional)

Logs:
```
2024-12-21T23:09:45+01:00 INFO Settings summary:
├── HTTP client
|   └── Timeout: 20s
├── Update
|   ├── Period: 10m0s
|   └── Cooldown: 5m0s
├── Public IP fetching
|   ├── HTTP enabled: yes
|   ├── HTTP IP providers
|   |   └── all
|   ├── HTTP IPv4 providers
|   |   └── all
|   ├── HTTP IPv6 providers
|   |   └── all
|   ├── DNS enabled: yes
|   ├── DNS timeout: 3s
|   └── DNS over TLS providers
|       └── all
├── Resolver: use Go default resolver
├── Server
|   ├── Listening address: :8000
|   └── Root URL: /
├── Health
|   └── Server listening address: 127.0.0.1:9999
├── Paths
|   ├── Data directory: ./data
|   ├── Config file: data/config.json
|   └── Umask: system default
├── Backup: disabled
└── Logger
    ├── Level: INFO
    └── Caller: hidden
2024-12-21T23:09:45+01:00 INFO reading JSON config from file data/config.json
2024-12-21T23:09:45+01:00 ERROR unknown provider: domeneshop
2024-12-21T23:09:45+01:00 INFO Shutdown successful
```

Configuration file (**remove your credentials!**):
```
{
  "settings": [
    {
      "provider": "domeneshop",
      "domain": "my.domain.no",
      "token": "***",
      "secret": "***",
      "ip_version": "ipv4",
      "ipv6_suffix": ""
    },
    {
      "provider": "duckdns",
      "domain": "my-domain.duckdns.org",
      "token": "***",
      "ip_version": "ipv4",
      "ipv6_suffix": ""
    }
  ]
}
```

Host OS: Linux Mint 21.3 x86_64

**Comment by hestadgard** [comment]
Everything works fine for duckdns when removing domeneshop from the config-file

**Comment by qdm12** [comment]
domeneshop was added in October 2024 and isn't part of the bug fix releases v2.8.1 or v2.8.2. You can build the binary from the master branch yourself (see readme) or wait for Christmas for a v2.9.0 release 🎄 🎅  thank you for your understanding & patient 🙏
